### ISSUE TITLE
Blacklisting mob types 

### ISSUE THREAD

**Issue by Ascacos** [Nathat23/StackMob-5#110]
Hi friend, I have a number of questions about this plugin:

Are we able to blacklist naturally spawned mobs from stacking? 
Is there a way to enable/disable mcmmo XP from stacked mobs?
Why do all the stacked mobs show up with the nametag "Spawned Mob", can i change/configure this?

Thanks for any assistance, hoping to have this properly configured as we updated our server to 1.16 recently and needed a new stacking plugin

Tyvm

**Comment by Nathat23** [comment]
1. Yes, add NATURAL to the "reason-blacklist"
2. Yes, using the option "mcmmo" in the "hooks section"
3. I don't believe that "Spawned Mob" would be cause by StackMob?

**Comment by Ascacos** [comment]
Is it also possible to add spawn-eggs as a reason blacklist?

**Comment by Nathat23** [comment]
That should be correct, did you make sure to restart the sever?

**Comment by Ascacos** [comment]
> That should be correct, did you make sure to restart the sever?

Yes I tried several things including /sm reload and restarting the server as a whole

**Comment by Ascacos** [comment]
I solved my issue by just inverting the reason blacklist to only allow SPAWNER reason

reason-blacklist: [SPAWNER]
reason-blacklist-invert: true

but might be something for you to look into why EGG isn't working

Much appreciation, your plugin is great
